### ISSUE TITLE
[BUG] XST download not possible

### ISSUE THREAD

**Issue by StephanMa** [eXist-db/xst#50]
> To be able to better understand you problem, please add as much information as possible to this ticket. Always test your bugs against the latest stable release of exist. We cannot provide support for older versions here on GitHub. If the version of eXist that is experiencing the issue is more than 1 major version behind the most recent release, please consider posting a question on our mailing list. 

**Describe the bug**
I try to download a resource/collection via xst. Unfortunately, I just get the xst splash screen.

**Expected behavior**
Download of the resource

**To Reproduce**
> The *best* way is to provide an [SSCCE (Short, Self Contained, Correct (Compilable), Example)](http://sscce.org/). One type of SSCCE could be a small test which reproduces the issue and can be run without dependencies. Please locate the existing test-suite and follow the examples provided there:

```
xst get /db/apps/stephan/ZZUXMPUX.xml . --config dockerhost
```
where "dockerhost" is the name of my config file. ls of collections works well... so I can confirm that my config file is working 

**Screenshots**
<img width="2533" alt="Bildschirm­foto 2022-12-20 um 15 49 25" src="https://user-images.githubusercontent.com/6735556/208694856-bf296327-f932-409e-99f6-7ebec400c45a.png">

**Context (please always complete the following information):**
 - OS: Ventura 13.0.1 MacOs
 - eXist-db Version: 5.2.0
-  App Version: XST build via npm install

**Additional context**
- How is eXist-db installed? started via bin/startup.sh
- Any custom changes in e.g. `conf.xml`? Nope

**Comment by line-o** [comment]
Node version would be helpful to know @StephanMa 
Will have to modify the issue template ;)

**Comment by line-o** [comment]
I have not tested node 19 yet. :eyes:

**Comment by line-o** [comment]
works for me
Build: eXist-6.1.0-SNAPSHOT (89109f644c19482a0f48f7d47619589ba2f14092)
Java: 1.8.0_352 (Azul Systems, Inc.)
Node: 19.3.0
OS: Mac OS X 12.6.2 (aarch64)

The behaviour you see is as if `xst` does not recognise the command at all. :/

**Comment by line-o** [comment]
@StephanMa would you be able to clone this repo, build xst locally and link it and then run the tests?
The detailed instructions are in [the development section of the readme](https://github.com/eXist-db/xst#development)

**Comment by line-o** [comment]
What format does your dockerhost config file have?

**Comment by line-o** [comment]
And what is the output of `xst --help`?

**Comment by line-o** [comment]
@StephanMa I just realised that the `get` command is missing in your screenshot!

**Comment by line-o** [comment]
It looks like you have an old version of `xst` installed.

What does `xst --version` give you?

**Comment by line-o** [comment]
the expected return of `xst --help` is

```
xst

interact with an exist-db instance

Commands:
  xst                                     interact with an exist-db instance                                                                                                                          [default]
  xst execute [<query>] [options]         execute a query in an exist-db instance                                                                                                          [aliases: run, exec]
  xst package <command>                   do something with packages                                                                                                                             [aliases: pkg]
  xst list [options] <collection>         List connection contents                                                                                                                                [aliases: ls]
  xst upload [options] <source> <target>  Upload files and directories to a target collection in exist-db                                                                                         [aliases: up]
  xst remove [options] <paths..>          remove collections and resources                                                                                                           [aliases: rm, delete, del]
  xst get [options] <source> <target>     Download a collection or resources from exist-db                                                                                           [aliases: download, fetch]

Options:
  --version  Show version number                                                                                                                                                                      [boolean]
  --config   configuration to use
  --help     Show help                                                                                                                                                                                [boolean]
```

**Comment by line-o** [comment]
But an old version 😄 

Try 

```
❯ npm unlink --global @existdb/xst
❯ npm install --global @existdb/xst
```

**Comment by line-o** [comment]
or you can also link it again, that's up to you

**Comment by line-o** [comment]
Since you also use a colored terminal the XST ascii logo should be bright yellow when calling `xst` without any commands

**Comment by line-o** [comment]
In your config you have
` "insert-final-newline": true`
and that is not supported in any released version of existdb. It will be released with exist 6.1.0.
